### PR TITLE
get Review by ID implemented, Links and routing added

### PR DESCRIPTION
**Pull request by Szpytma**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,7 +13,11 @@ function App() {
         name="NC Games"
         paragraph="No boredom on our board games."
       />
-      <ReviewList />
+      <Routes>
```
Comment (roisinoreilly) [inline src/App.js:17]: you've routed your reviews components to the right url but there's no link to these urls anywhere on your app, can you add them in please

File: src/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return isLoading ? (
+    <p>is loading...</p>
+  ) : (
+    <div className="Review">
+      <h2>{review.title}</h2>
+      <p>{review.review_body}</p>
+      <p>{review.category}</p>
+      <p>{review.designer}</p>
+      <p>{review.owner}</p>
+      <p>{review.created_at}</p>
```
Comment (roisinoreilly) [inline src/components/Review.jsx]: this date isn't formatted and isn't very user friendly, either manipulate it or install a package to do it for you (I like moment.js)

File: src/components/Review.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <p>{review.votes}</p>
+      <p>{review.comment_count}</p>
```
Comment (roisinoreilly) [inline src/components/Review.jsx]: all of these pieces of data could do with a description of what they are, at the moment there's just names and numbers with nothing showing the user what they are

File: src/components/ReviewCard.jsx [hunk 4]
```diff
@@ -1,9 +1,12 @@
+import { Link } from "react-router-dom";
 import "./ReviewStyles.css";
 function ReviewCard({ review }) {
   return (
     <div className="ReviewCard">
       <li key={review.review_id}>
-        <h4>{review.title}</h4>
+        <Link to={`/reviews/${review.review_id}`}>
+          <h4>{review.title}</h4>
```
Comment (roisinoreilly) [inline src/components/ReviewCard.jsx]: headings tags shouldn't be skipped eg. `<h1>` then `<h3>`, the next tag should be the next one down in the level for accessibility, they're not based on size. Can you double check all your h3 and h4 elements and make them h1/h2/h3s as needed. Don't forget to use WAVE/Lighthouse occasionally to check for issues like this

**Review by roisinoreilly — CHANGES_REQUESTED** [review]
a really great job so far :) check some of the comments I've left around accessibility, can you include a link to your reviews somewhere like the header and check your heading tags, once that's done send us another PR 👍

File: src/components/Welcome.jsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { Link } from "react-router-dom";
+
+function Welcome() {
+  return <div></div>;
```
Comment (RichardNeat) [inline src/components/Welcome.jsx]: Currently we aren't rendering any info on the home page and I have to manually update the URL to get to your reviews. Up to you how you would like this to work but we should be able to navigate your page without that manual update needed.

**Comment by Szpytma** [comment]
@roisinoreilly Thanks for the feedback. All is sorted now!

**Review by RichardNeat — APPROVED** [review]
Nice one Pawel - could you look into the comment below around your home page and also look into your responsive styling as it doesn't render well on mobile view currently. Lets try and stick to the mobile first approach. When that's done go ahead and merge no need to resubmit :)

**Pull request merged**